src/components/FeaturedPlantersSlider/index.js

```diff
@@ -16,11 +16,18 @@ import Link from '../Link';
 
 const SLIDE_EXTREME_INDEX = 30;
 
-function FeaturedPlantersSlider({ planters = [], color, link, size = null }) {
+function FeaturedPlantersSlider({
+  planters = [],
+  color,
+  link,
+  size = null,
+  isMobile,
```

Comment:
@Mohmn your update is good enough, but I think you didn't understand my idea, the word you are using is important for modularization, like the `language` used in the domain-driven design, instead of using `isMobile` you should use `hasSliderButtons`, I know they are the same in terms of implementing the function, but it brings different design for the model, the mobile or not is agnostic for this component (this is my original purpose ) the slider don't know and care about the mobile env, it just know these two buttons. 
So in the future, we might hide these two buttons for another reason, not because of mobile device. I think this is a better solution for building modules.

Reply:
Don't bother to change the name again, I will merge this now

Reply:
> @Mohmn your update is good enough, but I think you didn't understand my idea, the word you are using is important for modularization, like the `language` used in the domain-driven design, instead of using `isMobile` you should use `hasSliderButtons`, I know they are the same in terms of implementing the function, but it brings different design for the model, the mobile or not is agnostic for this component (this is my original purpose ) the slider don't know and care about the mobile env, it just know these two buttons. 
> So in the future, we might hide these two buttons for another reason, not because of mobile device. I think this is a better solution for building modules. 

Will keep that in mind from now on